string group_name
bool state
float32 rad
---